packages/admin-panel/src/editor/DataChangeAction.js

```diff
@@ -0,0 +1,29 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+import { connect } from 'react-redux';
+import { EDITOR_DATA_EDIT_BEGIN, EDITOR_DATA_EDIT_SUCCESS, EDITOR_ERROR } from './constants';
+
+const mapDispatchToProps = dispatch => ({
+  onEditBegin: () =>
```

Comment:
Minor: not sure what "begin" means for editing. The terms I've seen in form stuff is touched and dirty, maybe use one of those?